src/CoreAPIClient.ts

```diff
@@ -61,7 +61,7 @@ export default class CoreAPIClient {
         return this._get(apiUrl)
     }
 
-    private async _post(apiUrl: string, body: any): Promise<Response> {
+    private async _post(apiUrl: string, body: any, method = "POST"): Promise<Response> {
```

Comment:
When refactoring code default parameter values are usually troublesome. What's the reason for this change?

Reply:
well, it's the same as POST otherwise, so that keeps the code a bit more DRY I guess.

Reply:
If I refactor the test so that I don't need a PUT, I can revert that change for sure.

Reply:
My point is: Never use default values for function parameters, they cause trouble later on.

Reply:
ah well, in this case it saves the duplication of the complete function body. I don't disagree with that in general.

Reply:
Using default parameters don't save a complete function body.